### PR TITLE
Run codespell on source

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: test/e2e/pod_start_test.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ var _ = Describe("Podman pod start", func() {
 		session = podmanTest.Podman([]string{"pod", "start", podid1, podid2})
 		session.WaitWithDefaultTimeout()
 		// Different network backends emit different messages; check only the common part
-		Expect(session).To(ExitWithError(125, "ddress already in use"))
+		Expect(session).To(ExitWithError(125, "address already in use"))
```
Comment (edsantiago) [inline test/e2e/pod_start_test.go:101]: This one is intentional. Some errors use capital-A Address, some lower-case.

**Comment by edsantiago** [comment]
I'm tempted to leave the title as-is :-)

**Comment by rhatdan** [comment]
Fixed title as well.  Need recursive codespell.

**Review by giuseppe — APPROVED** [review]
LGTM

File: Makefile [hunk 2]
```diff
@@ -309,7 +309,7 @@ test/version/version: version/version.go
 
 .PHONY: codespell
 codespell:
-	codespell -S bin,vendor,.git,go.sum,.cirrus.yml,"*.fish,RELEASE_NOTES.md,*.xz,*.gz,*.ps1,*.tar,swagger.yaml,*.tgz,bin2img,*ico,*.png,*.1,*.5,copyimg,*.orig,apidoc.go" -L secon,passt,bu,hastable,te,clos,ans,pullrequest,uint,iff,od,seeked,splitted,marge,erro,hist,ether,specif -w
+	codespell -S bin,vendor,.git,go.sum,.cirrus.yml,"*.fish,RELEASE_NOTES.md,*.xz,*.gz,*.ps1,*.tar,swagger.yaml,*.tgz,bin2img,*ico,*.png,*.1,*.5,copyimg,*.orig,apidoc.go" -L ddress,secon,passt,bu,hastable,te,clos,ans,pullrequest,uint,iff,od,seeked,splitted,marge,erro,hist,ether,specif -w
```
Comment (edsantiago) [inline Makefile:312]: Is there a way to add magic comments, like for linters, immediately adjacent to the offending word? ISTM that we want to allow exactly one instance of `ddress`, but maybe not new ones?

Comment (rhatdan) [reply Makefile:312]: No idea, although I don't think this is a common spelling mistake.

**Comment by rhatdan** [comment]
@edsantiago mergeme?

**Comment by edsantiago** [comment]
/lgtm

Restarted yet another podman-machine flake

**Pull request merged**